presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -681,3 +681,81 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+
+Logging Properties
+------------------
+
+Presto uses the server application framework `Airlift
+<https://github.com/airlift/airlift>`_ and therefore supports its configuration
+properties for the HTTP server and general logging.
+
+
+``http-server.log.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``true``
+
+    Flag to enable or disable logging for the HTTP server.
+
+``http-server.log.compression.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``true``
+
+    Flag to enable or disable compression of the log files of the HTTP server.
+
+``http-server.log.path``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Default value:** ``var/log/http-request.log``
+
+    The path to the log file used by the HTTP server. The default is relative to
+    the Presto installation folder.
```

Comment:
it's relative to "data dir", which default to `<installation dir>/data`.

Can you please double check me by running Presto from unpacked tgz?

Reply:
yeah .. data dir..